fe/fe-core/src/main/java/com/starrocks/catalog/FsBroker.java

```diff
@@ -82,10 +82,10 @@ public boolean handleHbResponse(BrokerHbResponse hbResponse) {
             } else {
                 if (isAlive) {
                     isAlive = false;
-                    isChanged = true;
                 }
                 heartbeatErrMsg = hbResponse.getMsg() == null ? "Unknown error" : hbResponse.getMsg();
             }
+            isChanged = true;
```

Comment:
You should add some comment why every not-alive heartbeat will be marked changed.